api/v1alpha1/externaldns_types.go

```diff
@@ -421,37 +432,22 @@ type ExternalDNSSourceUnion struct {
 	// +required
 	Type ExternalDNSSourceType `json:"type"`
 
-	// AnnotationFilter describes an annotation filter
-	// used to filter which source instance resources
-	// ExternalDNS publishes records for.
-	// The annotation filter uses label selector semantics
-	// against source resource annotations.
+	// LabelFilter specifies a label selector used to filter which source instance resources ExternalDNS publishes
+	// records for. The filter uses label selector semantics against source resource labels.
```

Comment:
Does nil mean match all or match none.

Reply:
`nil` label filter means "filter nothing", so "get all source objects of the cluster".